src/Symfony/Component/HttpClient/Retry/HttpStatusCodeDecider.php

```diff
@@ -11,25 +11,56 @@
 
 namespace Symfony\Component\HttpClient\Retry;
 
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+
 /**
  * Decides to retry the request when HTTP status codes belong to the given list of codes.
  *
  * @author Jérémy Derussé <jeremy@derusse.com>
  */
 final class HttpStatusCodeDecider implements RetryDeciderInterface
 {
+    public const METHOD_DECISION = [
+        'GET' => true,
+        'HEAD' => true,
+        'PUT' => true,
+        'DELETE' => true,
+        'OPTIONS' => true,
+        'TRACE' => true,
+    ];
+    public const STATUS_CODES_DECISION = [
```

Comment:
Are we sure this should be public? This structure is very specific to this class. Maybe add a public static function to return a list of HTTP codes if we need to share them with the world.

Reply:
This were used by FrameworkBundle configuration. But you are right. This is internal.
Changed to private.